lib/deimos/active_record_consume/batch_record.rb

```diff
@@ -22,11 +22,12 @@ class BatchRecord
       # @param klass [Class < ActiveRecord::Base]
       # @param attributes [Hash] the full attribute list, including associations.
       # @param bulk_import_column [String]
-      def initialize(klass:, attributes:, bulk_import_column: nil)
+      # @param bulk_id_generator [Proc]
+      def initialize(klass:, attributes:, bulk_import_column: nil, bulk_id_generator: nil)
```

Comment:
Why is this sometimes called `bulk_id_generator` and sometimes `batch_id_generator`?

Reply:
Updated